query/src/sql/exec/mod.rs

```diff
@@ -362,6 +363,12 @@ impl PipelineBuilder {
             })?;
         }
 
+        // Since transform has added or did, making group expressions as column expr is safe.
+        group_expressions = group_expressions
```

Comment:
The group expression may have been a column, so there may be duplicate conversions here.